stuff/cron/testing/test_plagiarism_detector.py

```diff
@@ -0,0 +1,267 @@
+#!/usr/bin/env python3
+
+from typing import (List, Callable, Tuple, Iterable, Set, Dict)
+import pytest
+import dataclasses
+import datetime
+import string
+import time
+import itertools
+import sys
+import unittest
+import os
+import typing
+import random
+
+sys.path.insert(
+    0,
+    os.path.join(os.path.dirname(os.path.dirname(os.path.realpath(__file__))),
+                 "../"))
+
+import lib.db
+import cron.plagiarism_detector  # type: ignore
+
+# Constants
+
+_OMEGAUP_ROOT = os.path.abspath(os.path.join(__file__, '..'))
+LOCAL_DOWNLOADER_DIR = os.path.join(_OMEGAUP_ROOT, "testdata")
+
+SubmissionDownloader = Callable[[str, str, str], None]
+
+
+@dataclasses.dataclass()
+class Contest:
+    contest_id: int = 0
+    GUID: List[str] = dataclasses.field(default_factory=list)
```

Comment:
small nit:

```suggestion
    guids: List[str] = dataclasses.field(default_factory=list)
```

fields are normally lowercase. lists also are in plural.